engineering principles.md

```diff
@@ -1 +1,29 @@
 ## Engineering Principles at TotallyMoney
+
+### We Own It
+
+Ownership is the product of Autonomy, because if you have the freedom to choose you also are more likely to believe in your choices. It recognises the fact that we are a tech company, and we own the product.
+
+#### Practices
+
+- Know the customer
```

Comment:
Something, something.... We own the problem space and should know what problem we are trying to solve for the customer extremely well.

Reply:
Would work well in the description.